### PR TITLE
Allow for specifying a canonical url (to any page)

### PR DESCRIPTION
**Pull request by johanandren**
Canonical url is specified by adding canonical_url in the page preample

### CODE REVIEW

File: _includes/headertop.html [hunk 1]
```diff
@@ -9,6 +9,9 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <meta name="viewport" content="width=device-width" />
 
+    {% if page.canonical_url %}
+    <link rel="canonical" href="{{ page.canonical_url }}">
+    {% endif %}
```
Comment (patriknw) [inline _includes/headertop.html]: so simple when you know how it works, thanks

if I understand correctly we will just add `canonical_url: https://...` in the top section of the post md?

**Comment by ennru** [comment]
Isn't this very error-prone? Eg. when copying an md file and not updating the link?
There should be a way to create those...

File: _includes/headertop.html (hunk 1 above)
Comment (johanandren) [reply _includes/headertop.html]: correct

**Comment by johanandren** [comment]
@ennru It's added to define a specific canonical away from the site. In general we won't use it.

**Pull request merged**